docker/flags.go

```diff
@@ -48,6 +48,7 @@ var (
 		{"login", "Register or log in to a Docker registry server"},
 		{"logout", "Log out from a Docker registry server"},
 		{"logs", "Fetch the logs of a container"},
+		{"network", "Manages docker network"},
```

Comment:
network and service are experimental features. these must show up only on experimental builds.